storage/sqlite.go

```diff
@@ -370,6 +371,20 @@ func (s *Sqlite) updateUser(name string, r *bool, st *api.Status) error {
 		return err
 	}
 
+	tx := db.Updates(updates)
+
+	if tx.Error != nil {
+		return tx.Error
+	}
+
+	totalAffectedRows := tx.RowsAffected
+	if st != nil && totalAffectedRows == 1 {
+		updates["LastStatusUpdateTime"] = time.Now().Format(time.StampNano)
```

Comment:
db.UpdateColumn("last_status_update_time", time.Now())

don't use the Format method of the time package